util/spell.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Can you run https://www.shellcheck.net/ over this (locally or with the web version) to see if it passes with `#!/bin/sh`? If so, change the shebang so it works on any POSIX shell. If not, up to you whether we mandate bash (in which case change to `#!/usr/bin/env bash` since some OSes, notably NixOS, don't provide bash at the specified path as written) or fix  whatever shellcheck complains about to become POSIX compliant.

Due to how you use `$ASPELL` you'll almost certainly need to tell shellcheck to ignore some quoting heuristics even in Bash mode; go ahead and address those if you don't mind (or I can come in when I have a minute and do so).